containers/kubernetes/README.md

```diff
@@ -45,18 +44,20 @@ The ```openshift-template.json``` allows you to set the following environment va
 
 Only ```COCKPIT_KUBE_URL``` and ```OPENSHIFT_OAUTH_PROVIDER_URL``` are required.
 
-Deploy on kubernetes
---------------------
+Kubernetes Certificate Authority
+--------------------------------
 
-This container will only work with kubernetes if the API is open or configured to use basic authentication. See http://kubernetes.io/v1.1/docs/admin/authentication.html for more information.
+The container communicates with the kubernetes API over SSL. By default kubernetes uses a self signed certificate. You can tell Cockpit what the correct certificate authority is by setting
+the contents of KUBERNETES_CA_DATA. Cockpit will also look for ca.crt in the default service account seceret that kubernetes will mount in the container by default. If you don't want cockpit to verify the certificate authority when connecting to the API you can set ```KUBERNETES_INSECURE``` to ```true```
 
-WARNING: If you choose to deploy this container to communicate with an open kubernetes API you should take care to not expose it to an insecure network.
+Web Certificates
+----------------
 
-```
-kubectl create -f deploy-examples/cockpit-kubernetes.json
-```
+Cockpit looks at ```/etc/cockpit/ws-certs.d/``` to find its SSL certificate. You should create a Kubernetes secret API object to place your PEM encoded certificates and key in a ```.cert``` file in that location.
 
-This will create a cockpit-kube service, replication controller and pod.
+If there are no certificates present in that location cockpit will generate and use a self-signed certificate. However the CN will be based on the container name so this option should be avoided for most deployments.
+
+You can also choose to serve cockpit without SSL by setting the ```COCKPIT_KUBE_INSECURE``` environment variable to "true"
```

Comment:
missing `.` and [here](https://github.com/cockpit-project/cockpit/pull/3972/files#diff-2c15a0b2bf7087918fc14e83a8b67dddR51) you wrote ```true``` instead of "true"